### PR TITLE
Added rank question type, closes #135

### LINKED ISSUE #135: add rank type

### PR DESCRIPTION
**Pull request by MartijnR**

### CODE REVIEW

File: _sections/home-english.md [hunk 1]
```diff
@@ -171,6 +172,30 @@ Click on the link to look at the complete [pizza_questionnaire](https://docs.goo
 **Caveat**  
 When you export data using this **or_other** option, in the **favorite_topping** column, you will see a value **other**. A separate column will have the answer for the questions in which the user selected **other**. This makes data analysis more cumbersome, so we do not recommend the **or_other** construct for large scale data collection efforts. See the **Relevant** section below for an alternative method more appropriate for large scale projects.
 
+### Rank
+
+The rank widget can be use to let respondents order a list of options. The answer is saved as an ordered space-separated list of option values where all options are always included. The syntax is very similar to multiple-choice questions.
```
Comment (jnm) [inline _sections/home-english.md]: Typo:
> can be use[d]

Minor style suggestion:
> saved as an ordered[,] space-separated list

File: _sections/home-english.md [hunk 2]
```diff
@@ -171,6 +172,30 @@ Click on the link to look at the complete [pizza_questionnaire](https://docs.goo
...
+
+| type                    | name             | label                                         |
+| ----------------------- | ---------------- | --------------------------------------------- |
+| rank pizza_toppings     | toppings         | Order pizza topping with your favorite on top |
```
Comment (jnm) [inline _sections/home-english.md]: > Order pizza topping[s]

File: _sections/home-english.md [hunk 3]
```diff
@@ -171,6 +172,30 @@ Click on the link to look at the complete [pizza_questionnaire](https://docs.goo
...
+| ======================= | ============     | ============================================= |
+| survey                  |                  |                                               |
+
+<p/>
+
+| list name      | name       | label                     |
+| -------------- | ---------- | ------------------------- |
+| list name      | name       | label                     |
```
Comment (jnm) [inline _sections/home-english.md]: Duplicate header row?

File: _sections/home-english.md (hunk 2 above)
Comment (MartijnR) [reply _sections/home-english.md]: Yes, 1 topping no pizza make.

**Comment by MartijnR** [comment]
Thanks! 👍

**Pull request merged**